app/Http/Controllers/Frontend/AirportController.php

```diff
@@ -36,8 +36,8 @@ public function show($id, Request $request)
     {
         $id = strtoupper($id);
 
-        $airport = $this->airportRepo->find($id);
-        if (empty($airport)) {
+        $airport = $this->airportRepo->where('id', $id)->first();
+        if (!$airport) {
```

Comment:
Weird, this one should work

Reply:
Well, I expected it to be working too ... Realized that it was not when I entered a wrong icao to address bar.